.changeset/tidy-years-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/image": minor
```

Comment:
Change to `patch`

Reply:
@108yen 

Thanks! Is the changed file ok?

Reply:
@Eiji-Kudo 
Perfect!